packages/db-base/src/lib/inMemFileDB.ts

```diff
@@ -110,29 +173,29 @@ class InMemoryFileDB {
     /**
      * Loads the dataset including pot. Fallback handling
      *
-     * @param datasetName {string} Filename of the file to load
-     * @returns {Promise<Record<string, any>>} dataset read as object
+     * @param datasetName Filename of the file to load
+     * @returns obj dataset read as object
      */
-    async loadDataset(datasetName) {
+    async loadDataset(datasetName: string): Promise<Record<string, any>> {
         let ret = {};
         try {
             ret = await this.loadDatasetFile(datasetName);
 
             // loading worked, make sure that "bak" File is not broken
             try {
                 await fs.readJSON(`${datasetName}.bak`);
-            } catch (e) {
+            } catch (e: any) {
```

Comment:
If we're doing this a lot, it might make sense to set this to false in tsconfig instead:
https://www.typescriptlang.org/tsconfig#useUnknownInCatchVariables

Reply:
is there a better practice than using `any`?

Reply:
If the option is `false`, the error variables default to `any`. If it is `true` (changed in some 4.x version), they default to `unknown`, which is safer but a bit more complicated to work with.

IMO disabling the option and removing the (then unnecessary) `: any` annotations is fine.

Reply:
will think about to change this for porting the next module